README.md

```diff
@@ -11,15 +11,18 @@ Modify the iOS Simulator so that it has a perfect status bar, then run your app
 
 Starting in Xcode 11, the `simctl` command line tool includes a `status_bar` option that allows you to override the appearance of the status bar in the simulator. Hopefully this will eventually supercede the need for SimulatorStatusMagic, but at the moment it still has holes that make this project continue to be relevant. In particular, `simctl status_bar` does not currently provide a way to add localized date and time strings in the status bar.
 
-## 1) Injecting into Springboard (Required on iOS 17+)
-**tl;dr** Running `build_and_inject.sh booted` will apply a default status bar to the running simulator. Replace "booted" with a simulator UDID to target a specific simulator.
+## Usage
+
+### 1) Injecting into Springboard (Required on iOS 17+)
+**tl;dr** Running `build_and_inject.sh booted` will apply a default status bar to the running simulator. Replace "booted" with a simulator UDID to target a specific simulator. Retrieve them with `xcrun simctl list devices | grep 'Booted'`
+
+**For XCTest simulators, use `build_and_inject_test.sh booted`** - Replace "booted" with a simulator UDID to target a specific simulator. Retrieve them with `xcrun simctl --set testing list devices | grep 'Booted'`
```

Comment:
Nice, I think this is a great addition.

We should also clarify that this is only for duplicated simulators created by using the `-parallel-testing-enabled` flag on xcodebuild. If you are using XCTest but not using parallel testing, this line might be misleading.